docs/src/main/asciidoc/spring-stream.adoc

```diff
@@ -75,13 +75,24 @@ spring.cloud.stream.gcp.pubsub.bindings.{CONSUMER_NAME}.consumer.ack-mode=AUTO_A
 If automatic resource creation is turned ON and the subscription and/or the topic do not exist for a consumer, a subscription and potentially a topic will be created.
 The topic name will be the same as the destination name, and the subscription name will be the destination name followed by the consumer group name.
```

Comment:
Since you talk about the subscription name below, maybe take out the second part of the sentence?

Reply:
removed